deep_qa/common/checks.py

```diff
@@ -30,9 +30,5 @@ def log_keras_version_info():
         backend = K.backend()
     except AttributeError:
         backend = K._BACKEND  # pylint: disable=protected-access
-    if backend == 'theano':
```

Comment:
Actually, can you keep this here?  Instead of removing these lines, add one after the old line 35, that looks like this:

```python
logger.warning("Using Keras' theano backend is not supported! Expect to crash...")
```